### DISCUSSION TITLE
How to define the power-on position ?

### DISCUSSION THREAD

**Discussion by santonum** [Dlloydev/ESP32-ESP32S2-AnalogWrite#37] [Q&A]
Hi, 
Using 0.x easing, if my servo is powered with a 40° position, and if **the first position** to go is 80°, the servo will first move to 0°C and then move up to 80°, because the PWM power-on position is 0°, and it calculates its way from this origin.
For the **second** move, it has stored the good position at 80°, so if I drive it to go to 120°, it's fine, it won't start by going to 0.

So is there a way to set the power-on PWM position before the first PWM move ?
I've got a servo with feedback signal, so I can know the power-on position to tell to the PWM algorithm.
Thanks

**Comment by Dlloydev** [comment]
Sure, at the beginning of setup(), use something like this: `myservo.write(servoPin, 40);`
This would attach the servo to the first available channel and set the position to 40 degrees.

You could use attach and specify other parameters if needed, but follow it up with a simple write function as above so the initial degrees would be set immediately and become your starting point.
